content/pages/exec-S15.md

```diff
@@ -0,0 +1,40 @@
+Title: Spring 2015 Committee
```

Comment:
This should actually be made on the exec.md file. I'm just going to quickly redo this commit based on the info here.